### ISSUE TITLE
Support to asynchronous register an icon

### ISSUE THREAD

**Issue by VitorLuizC** [MMF-FE/vue-svgicon#23]
I'm doing code-split with my application icons, so I've changed the default template and in usage I've noticed that I can't asynchronous register an icon.

```js
import { register } from 'vue-svgicon';

import icon from './icons/engine';    // Okay
register({ [icon.name]: icon.data }); //

import('../icons/trash')                                              // Fail
  .then(({ default: icon }) => register({ [icon.name]: icon.data })); // 
```

**Comment by VitorLuizC** [comment]
I've made a component that wrap SVGIcon and only show icon after import it. This fixed this issue, but vue-svgicon should have something to deal with asynchronous registered icons.

```vue
<template>
  <icon v-if="!loaded" class="c-icon" />
  <icon v-else class="c-icon" :icon="icon" :width="width" :height="height" />
</template>

<script>
  import Icon from 'vue-svgicon'

  export default {
    components: { Icon },
    data: () => ({
      loaded: false
    }),
    props: {
      width: [Number, String],
      height: [Number, String],
      icon: {
        type: String,
        required: true
      }
    },
    mounted () {
      import('../icons/' + this.icon)
        .then(() => {
          this.loaded = true
          console.log(`"${this.icon}" icon was loaded...`)
        })
        .catch(error => {
          this.loaded = false
          console.error(error)
        })
    }
  }
</script>
```

**Comment by Allenice** [comment]
@VitorLuizC Fixed in v1.2.6
